### PR TITLE
Feature/61

### PR DESCRIPTION
**Pull request by rodriguez-facundo**
Add get_credentials to dataSourceService for password protected db

### CODE REVIEW

**Comment by rodriguez-facundo** [comment]
@filippomc

File: pygeppetto/model/datasources/datasources.py [hunk 1]
```diff
@@ -144,6 +144,7 @@ class DataSource(Node):
     dependenciesLibrary = EReference(ordered=True, unique=True, containment=False, upper=-1)
     targetLibrary = EReference(ordered=True, unique=True, containment=False)
     fetchVariableQuery = EReference(ordered=True, unique=True, containment=True)
+    auth = EAttribute(eType=EString, derived=False, changeable=True)
```
Comment (filippomc) [inline pygeppetto/model/datasources/datasources.py:147]: @rodriguez-facundo this is not a change we can make here as this code is automatically generated from the ecore specification.  Usually those changes are made by @tarelli in the Java side.

**Review by filippomc — CHANGES_REQUESTED** [review]
@rodriguez-facundo the approach looks good and the code is clean. 
Before really committing that to pygeppetto we must be sure that there are no other ways already to use credentials we overlooked and implement this if needed through the ecore specification.

**Comment by filippomc** [comment]
@facundo we just need to replace the file geppettoModel.ecore with this https://github.com/openworm/org.geppetto.model/blob/de10cc313f4e79b37e2385266f630cd16dd58a6c/src/main/resources/geppettoModel.ecore , then we are ready to merge and we'll release pygeppetto 0.8.1 with your changes. cc @tarelli

**Comment by rodriguez-facundo** [comment]
@filippomc updated

**Review by filippomc — APPROVED** [review]

**Pull request merged**